package.json

```
@@ -77,6 +77,11 @@
         "one-vscode.enableCodelens": {
           "type": "boolean",
           "default": true
+        },
+        "one-vscode.projectRootDir": {
```

```suggestion
        "one-vscode.projectRootDirectory": {
```

Is it acceptable or welcome to use abbreviation in user description? (I'm not familiar with making user description.)

---

Good point. I checked and found that `Path` is commonly used. But, yes, I like `Directory` since it is more specific. (`Path` may mean path of a file)